src/components/html/editForm/HTML.edit.display.js

```diff
@@ -59,4 +59,19 @@ export default [
     key: 'refreshOnChange',
     input: true
   },
+  {
+  type: 'select',
```

Comment:
Spacing is incorrect.
Also, can we just make this a Checkbox so that the setting is a boolean instead of a string?

Reply:
@travist  check now

Reply:
@travist  it's ok ?